charts/scalar-manager/values.yaml

```diff
@@ -80,12 +84,65 @@ api:
+    authentication.providers.static-jwt.public-key=${AUTHENTICATION_PROVIDERS_STATIC_JWT_PUBLIC_KEY:}
+    authentication.providers.static-jwt.issuer-uri=${AUTHENTICATION_PROVIDERS_STATIC_JWT_ISSUER_URI:https://scalar-manager.scalar-labs.com}
+    authentication.providers.static-jwt.access-token-expiration-time=${AUTHENTICATION_PROVIDERS_STATIC_JWT_ACCESS_TOKEN_EXPIRATION_TIME:15m}
+    authentication.providers.static-jwt.refresh-token-expiration-time=${AUTHENTICATION_PROVIDERS_STATIC_JWT_REFRESH_TOKEN_EXPIRATION_TIME:3d}
+
+    # OpenAPI configuration
+    springdoc.swagger-ui.enabled=${SPRINGDOC_SWAGGER_UI_ENABLED:false}
+    springdoc.swagger-ui.path=${SPRINGDOC_SWAGGER_UI_PATH:/swagger-ui.html}
+
+    # Whether to enable persistence endpoints or not (auth, user management)
+    app.persistence-endpoints.enabled=${APP_PERSISTENCE_ENDPOINTS_ENABLED:false}
+
+    # Whether to enable authorization or not for the operational endpoints
+    app.authorization.enabled=${APP_AUTHORIZATION_ENABLED:false}
+
+    # CORS configuration
+    app.cors.allowed-origins=${APP_CORS_ALLOWED_ORIGINS:*}
+    app.cors.allowed-methods=${APP_CORS_ALLOWED_METHODS:*}
+    app.cors.allowed-headers=${APP_CORS_ALLOWED_HEADERS:*}
+
+    # # Initial admin configuration, only need these configurations if persistence endpoints is enabled
+    # app.initial-admin-user.enabled=${APP_INITIAL_ADMIN_USER_ENABLED:false}
+    # app.initial-admin-user.email=${APP_INITIAL_ADMIN_USER_EMAIL:admin@scalar.com}
+    # app.initial-admin-user.name=${APP_INITIAL_ADMIN_USER_NAME:Administrator}
+    # app.initial-admin-user.password=${APP_INITIAL_ADMIN_USER_PASSWORD:Password@123!}
+
+    # # JPA configuration, only need these configurations if persistence endpoints is enabled
+    # spring.jpa.hibernate.ddl-auto=${SPRING_JPA_HIBERNATE_DDL_AUTO:update}
+    # spring.jpa.show-sql=${SPRING_JPA_SHOW_SQL:false}
+    # spring.jpa.properties.hibernate.format_sql=${SPRING_JPA_PROPERTIES_HIBERNATE_FORMAT_SQL:false}
+
+    # # Database configuration, only need these configurations if persistence endpoints is enabled
+    # spring.datasource.url=jdbc:postgresql://${DATABASE_HOST:scalar-manager-postgres-postgresql}:${DATABASE_PORT:5432}/${DATABASE_NAME:scalar-manager}
+    # spring.datasource.username=${DATABASE_USERNAME:scalar-manager}
+    # spring.datasource.password=${DATABASE_PASSWORD:scalar-manager}
+    # spring.datasource.driver-class-name=org.postgresql.Driver
+
+
 web:
   image:
     repository: ghcr.io/scalar-labs/scalar-manager-web
     pullPolicy: IfNotPresent
     # Overrides the image tag whose default is the chart appVersion.
     tag: ""
+
+  authorization:
+    # Whether to enable authorization or not for the web application, if enabled the login, user management page will be available
+    enabled: false
+    # The base URL of the authorization service, default is same as the scalar-manager-api service
+    baseUrl: http://localhost:8080
```

Comment:
In the case of the API service running in the same pod with Scalar Manager web doesn't have the persistent endpoints enabled then we can config this `baseUrl` point to another instance of the Scalar Manager API (in other clusters) to do authentication. Note that the JWT public key need to be configured the same in the API instances